src/setup/helper.js

```diff
@@ -9,80 +9,90 @@ Please make sure you have installed following dependencies:
 
 * Git
 * Node/npm v10.17.0 (or higher)
-* Go 1.13+
+* Go 1.18+
 * Rabbitmq (Latest stable version)
 * Solc v0.5.11 (https://solidity.readthedocs.io/en/v0.5.3/installing-solidity.html#binary-packages)
 * Ganache CLI (https://www.npmjs.com/package/ganache-cli)
 `))
 }
 
 export async function getChainIds(options = {}) {
-  const questions = []
-
-  if (!options.borChainId) {
-    questions.push({
-      type: 'input',
-      name: 'borChainId',
-      message: 'Please enter Bor chain id',
-      default: '15001'
-    })
-  }
+  // const questions = []
 
-  if (!options.heimdallChainId) {
-    questions.push({
-      type: 'input',
-      name: 'heimdallChainId',
-      message: 'Please enter Heimdall chain id',
-      default: 'heimdall-15001'
-    })
+  return {
+    borChainId: '15001',
+    heimdallChainId: 'heimdall-15001'
   }
 
-  // return if no questions
-  if (questions.length === 0) {
-    return {}
-  }
-
-  // get answers
-  return await inquirer.prompt(questions)
+  // if (!options.borChainId) {
```

Comment:
Remove comments?

Reply:
We have disabled it for now.. Intentionally did not remove the comments. @vcastellm